### ISSUE TITLE
Time Presets - running Cacti 1.1.38. Unable to zoom or see the past 4 hours of graphs.

### ISSUE THREAD

**Issue by Talria** [Cacti/cacti#3823]
![image](https://user-images.githubusercontent.com/71720855/94008751-8142ec80-fd71-11ea-995a-b255fb317565.png)

I've changed both php files from 1600000000 to 2600000000 and restarted the server, and still doesn't allow me to zoom into the graph, or see any graph in the past 4 hours. :( Anything else I can try??

![image](https://user-images.githubusercontent.com/71720855/93909780-27d5b180-fcce-11ea-83e8-ac69cc0b07c1.png)
![image](https://user-images.githubusercontent.com/71720855/93909904-53589c00-fcce-11ea-8bdb-e599c9b1938d.png)

_Originally posted by @Talria in https://github.com/Cacti/cacti/issues/3797#issuecomment-696830429_

**Comment by toobadsosad** [comment]
i think this not the same problem.. you sure your poller is running?
the timestamp bug is only affecting zooming graph.. not making graphs not drawing

**Comment by cigamit** [comment]
Seems more like a timezone issue.

**Comment by gaankdoank** [comment]
> 
> 
> graph_image.php and graph_json.php?
> 
> but i think this not the same problem.. you sure your poller is running?
> the timestamp bug is only affecting zooming graph.. not making graphs not drawing
> 
> as i noticed that the left side of your graphs is blank?

i have the same issue with @Talria , what timestamp do you mean ? i've already adjusting system datetime and DB server but no changes appear.

**Comment by TheWitness** [comment]
If you Cacti system is not in UTC, in the old days, when you bounce in/out of Daylight Savings, you would get a gap or miss one hours worth of data a year.  I thought we had worked around that, but if someone is finding that it's not working, maybe @browniebraun can incorporate that in his current project.

Otherwise check that your poller is running.

**Comment by TheWitness** [comment]
He meant, did you update both files....

**Comment by gaankdoank** [comment]
> 
> 
> He meant, did you update both files....

@toobadsosad thank you for your information, this solved by edited both files.

**Comment by gustavc39** [comment]
Faced the same issue, fixed by editing the two files stated above. Thanks!

**Comment by upengan78** [comment]
How was this issue resolved by upstream ? Are users of expected to resolve issues by editing the php files or it will be fixed in the package? I am using 1.1.38 from Ubuntu repositories and values are still set to 1600000000 and I could not zoom any graphs until I updated the php files on Nov 15 2020. Also previous month did not work until I changed the php files. This is really weird problem. One might think they did something to screw this up .lol
